### ISSUE TITLE
cycle_momentum option for OneCycle learning rate scheduler

### ISSUE THREAD

**Issue by MarvinCARIAD** [open-mmlab/mmcv#2645]
### What is the feature?

Could similar, to the PyTorch implementation, a "cycle_momentum" option be added to the OneCycle learning rate scheduler?
Documentation of PyTorch implementation: https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.OneCycleLR.html

Alternatively, if there is another way to achieve the same result with the current implementation, then it would be great to know how to do this.
